### ISSUE TITLE
[UI] Mouse cursor vanishing

### ISSUE THREAD

**Issue by Mystler** [SimCMinMax/WoW-BugTracker#12]
**Game Version:**
> 8.0.1 Alpha Build 25976

**Description:**
> Mouse cursor vanishes and you can only get it back by restarting the game.

**Details:**
> I don't know how this bug is triggered, but on occasion the mouse cursor gets invisible and you have to restart the client. Alt-tabbing out and back in gave me the Windows cursor for a few seconds until it vanished again.

**Reproducible:**
> Examples: rarely

**Howto:**
> Not sure.

**Comment by Aethys256** [comment]
I experienced it two times aswell, I do play with Windowed Fullscreen and use G-Sync (if that matters).
For the rest, pretty standard setup I guess (don't think it matters).

Happened to @Nianel aswell. We both uses Razer Naga with Razer Synapse 2.0.

Maybe you should post your setup aswell @Mystler.

**Comment by Mystler** [comment]
Good idea. I also use a Razer Naga and have the game in Windowed Fullscreen.

**Comment by Zarania** [comment]
Happened to me, no razor naga.  Windowed fullscreen

**Comment by Hinalover** [comment]
Happened to me while finishing up the frog quests. Do not have a Razor Naga, and using in Window Mode. Moving the mouse out of the game and back in temporarily gives a cursor, but refreshing the game provides a temporary fix.
